docs/conf.py

```diff
@@ -57,7 +57,7 @@
 # The short X.Y version.
 version = '0.11.5'
 # The full version, including alpha/beta/rc tags.
-release = '0.11.5'
+release = '0.11.5rc6'
```

Comment:
this one and setup.py are on rc6 while version() returns v0.11.5. Is this intentional? (if so; fine)

Reply:
right so we had a bit of back and forth about include or don't include. Since the comment specifically mentions the tags I decided to keep the RC designation